docs/docs/operator-manual/setup-guide/operator.mdx

```diff
@@ -0,0 +1,117 @@
+import {
+RIG_OPERATOR_CHART_VERSION
+} from "../../../src/constants/versions"
+
+# Rig Operator
+
+## Setup guide
+
+Installing the Rig Operator is a easy as any other Kubernetes operator. We've made a
+Helm chart available the makes it easy to get all the resources up and running.
+
+## Dependencies
+
+The rig-operator uses the kubernetes APIs and a few additional APIs to provide
+some very easy-to-use APIs for spinning up workloads.
+
+- [cert-manager](https://cert-manager.io/).
+- [metrics-server](https://github.com/kubernetes-sigs/metrics-server).
+
+### cert-manager
+
+The rig-operator helm chart uses cert-manager to create a certificate for the
+CRD webhooks. These are self-signed and doesn't require any special
+configuration of cert-manager.
+
+If you want to enable ingress support, then you will need to further configure
+cert-manager and rig-operator to be able to generate certificates for
+workloads. See [the section about ingress](/operator-manual/ingress) for more
+information about this.
```

Comment:
Should we likewise have a small command checking if `cert-manager` is installed as for the metrics server?

Reply:
Yeah, that would be nice

Reply:
done.